src/components/ProofOfAddress/PoAGuidance/graphic.js

```diff
@@ -1,4 +1,3 @@
-import { h } from 'preact'
 import style from './style.scss'
```

Comment:
Not so sure for this case, since it's just rendering an SVG but the SDK components tend to require the `import { h } from 'preact'` to work.

Reply:
As the h is not used in the code, but still required, would it make sense to follow https://stackoverflow.com/questions/52417272/import-h-from-preact-when-is-it-necessary